### DISCUSSION TITLE
Adjust the color conversion

### DISCUSSION THREAD

**Discussion by beyondthemind91an** [krojew/OpenRGB-Ambient#5] [Ideas]
First of all, thank you for this plugin! I use it daily and it works excellent!

For me the ambient light is still a bit to cold. I'm using a homebuilt controller together with WS2812B leds. Would it be possible to add something that makes the color temperature adjustable?

Thank you!

**Comment by krojew** [comment]
Done :) https://github.com/krojew/OpenRGB-Ambient/releases/tag/2.2.0
